packages/sdk/lib/base.ts

```diff
@@ -182,6 +200,8 @@ const DEFAULT_PROD_OPTIONS: Required<Options> = {
   validation: Validation.TrackOnInvalid,
 };
 
+type ReturnedPromise = { promise: Promise<void> };
```

Comment:
`CallResponse` might be a better name. That would allow us to add other values in the future. What do you think?

Reply:
Agreed